micromasters/utils.py

```diff
@@ -62,3 +63,20 @@ def custom_exception_handler(exc, context):
             status=status.HTTP_500_INTERNAL_SERVER_ERROR,
             data=[formatted_exception_string]
         )
+
+
+def serialize_model(obj):
+    """
+    Serialize model into a dict representable as JSON
+
+    Args:
+        obj (django.db.models.Model): An instantiated Django model
+    Returns:
+        dict:
+            A representation of the model
+    """
+    # serialize works on iterables so we need to wrap object in a list, then unwrap it
+    data = json.loads(serialize('json', [obj]))[0]
+    fields = data['fields']
```

Comment:
nit: 'fields' is only really relevant as a key name in the return value of `serialize`. `serialized` or `serialized_obj` is probably more on the mark for this variable